include/wpe/input.h

```diff
@@ -93,6 +93,8 @@ enum wpe_input_axis_event_type {
     wpe_input_axis_event_type_null,
     wpe_input_axis_event_type_motion,
     wpe_input_axis_event_type_motion_smooth,
+
+    wpe_input_axis_event_type_mask_2d = 1 << 16,
```

Comment:
type_mask_2d sounds to me like mask is the axis event type, maybe type_2d_mask which is the mask for the 2d type?